### ISSUE TITLE
Use Fluent Bit community version on EC2 instances

### ISSUE THREAD

**Issue by JoeShi** [aws-solutions/centralized-logging-with-opensearch#256]
### Describe the feature

By switching from Fluent Bit AWS distribution to the community distribution, you no longer need to install OpenSSL v1.1.1 or above on your instances.

### Use Case

I am frustrated wo install the OpenSSL v1.1.1 on instances. not all instances by default shipped with OpenSSL v1.1.1 or above.

### Solution version used

v2.1.1

**Comment by adiospeds** [comment]
Yes pls make this change -- It makes it difficult to deploy to standalone instances and thus is causing a lot of manual steps for us.

**Comment by adiospeds** [comment]
Hi team .. request you to look into this on priority if possible .. since older version is creating more and more compatibility problems
